README.Rmd

```diff
@@ -22,7 +22,7 @@ knitr::opts_chunk$set(
 [![CRAN status](https://www.r-pkg.org/badges/version/finalsize)](https://CRAN.R-project.org/package=finalsize)
 <!-- badges: end -->
 
-_finalsize_ provides calculations for the final size of an epidemic in a population with demographic variation in contact patterns, and variation within and between age groups in their susceptibility to disease.
+_finalsize_ provides calculations for the final size of an epidemic in a population, which is the overall number of infected individuals, depending on a demographic distribution (e.g., age groups) and their contact patterns, accounting for different susceptibility to disease between groups (e.g., due to age-related impaired or dysregulated immune responses) and within groups (e.g., due to immunization programs).
```

Comment:
Thanks for updating this. However, after reviewing more about how "susceptibility" is defined as "the probability of acquiring infection when exposed" in math modelling papers about the age effects on transmission [[1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9000131/),[2](https://www.nature.com/articles/s41591-020-0962-9)], I prefer to use suggest a more appropriate line.

I suggest this:
"between groups (e.g. due to age-specific exposure to similar pathogens conferring immune protection)"

instead of:
"between groups (e.g., due to age-related impaired or dysregulated immune responses)"

which is more related to "clinical susceptibility".

Reply:
I think we can stick to a more general description of the mechanism, such as "age-related effects on immune function", and add the references all together. Acquiring an infection on exposure probably depends on a wide range of immune system mechanisms, and age is likely to influence these in different ways. I guess our users will understand that this is a general model and can be interpreted in many ways.

Reply:
Updated in https://github.com/epiverse-trace/finalsize/pull/121/commits/efe38353f391122c8c4efdbe71221a6b43a1a003